src/components/Superscript.js

```diff
@@ -15,10 +14,13 @@ import {
   pathToContext,
 } from '../util'
 
+// selectors
+import { exists } from '../selectors'
+
 const mapStateToProps = ({ contextViews, cursor, cursorBeforeEdit, modalData, showModal }, props) => {
 
   // track the transcendental identifier if editing
-  const editing = equalArrays(pathToContext(cursorBeforeEdit || []), pathToContext(props.thoughtsRanked || [])) && exists(headValue(cursor || []))
+  const editing = equalArrays(pathToContext(cursorBeforeEdit || []), pathToContext(props.thoughtsRanked || [])) && exists(store.getState().thoughtIndex, headValue(cursor || []))
```

Comment:
store.getState() is similar to the first argument of mapStateToProps, which is being de-structured. So you can use that here and pass state to `exists` function